app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -2146,6 +2149,8 @@ class BrowserTabFragment :
                         when (method) {
                             "webShare" -> if (id != null && data != null) { webShare(featureName, method, id, data) }
                             "permissionsQuery" -> if (id != null && data != null) { viewModel.onPermissionsQuery(featureName, method, id, data) }
+                            "screenLock" -> if (id != null && data != null) { screenLock(featureName, method, id, data) }
```

Comment:
I believe is time to move this when to the view model ala `SubscriptionsWebViewActivity`